File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.swift
import Foundation

func solution(_ num_list:[Int], _ n:Int) -> [[Int]] {
    var answer: [[Int]] = []
    for i in stride(from: 0, to: num_list.count, by: n) {
        var list: [Int] = []
        for j in 0..<n {
            list.append(num_list[i + j])
        }
        answer.append(list)
    }
    return answer
}

File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.js
function solution(num_list, n) {
    const answer = [];
    let array = [];
    for (index in num_list) {
        array.push(num_list[index]);
        console.log(array)
        if (index % n === n - 1) {
            answer.push(array);
            array = [];
        }
    }
    return answer;
}